src/main/resources/ui/controller/UpsertWorkflow.controller.js

```diff
@@ -186,7 +199,8 @@ sap.ui.define([
 
 		sensorTypes: [
 			{name: "Absa-Kafka", fragment: "absaKafka", controller: "AbsaKafkaController"},
-			{name: "Kafka", fragment: "kafka", controller: "KafkaController"}
+			{name: "Kafka", fragment: "kafka", controller: "KafkaController"},
+			{name: "Time", fragment: "time", controller: "TimeController"}
```

Comment:
This should be slowly moved to configuration

Reply:
Will be fixed here: #60